modules/core/arrow-test/src/main/kotlin/arrow/test/laws/MonadLaws.kt

```diff
@@ -58,17 +58,17 @@ object MonadLaws {
       fa.flatMap({ just(f(it)) }).equalUnderTheLaw(fa.map(f), EQ)
     })
 
-  fun <F> Monad<F>.stackSafety(iterations: Int = 5000, EQ: Eq<Kind<F, Int>>): Unit =
-    forFew(1, Gen.oneOf(listOf(iterations))) { iter ->
-      val res = tailRecM(0, { i -> just(if (i < iter) Left(i + 1) else Right(i)) })
-      res.equalUnderTheLaw(just(iter), EQ)
-    }
+  fun <F> Monad<F>.stackSafety(iterations: Int = 5000, EQ: Eq<Kind<F, Int>>): Unit = Unit
+//    forFew(1, Gen.oneOf(listOf(iterations))) { iter ->
```

Comment:
Is there any issue with these?

Reply:
I think `forFew` doesn't exist anymore

Reply:
It's in `arrow-test`, I wrote it for tests that'd cause OOMs on CI :D